src/kit/taosdemo/taosdemo.c

```diff
@@ -2663,117 +2663,130 @@ static int printfInsertMeta() {
         }
 
-        printf("  super table count:     \033[33m%"PRIu64"\033[0m\n",
-                g_Dbs.db[i].superTblCount);
-        for (uint64_t j = 0; j < g_Dbs.db[i].superTblCount; j++) {
-            printf("  super table[\033[33m%"PRIu64"\033[0m]:\n", j);
 
-            printf("      stbName:           \033[33m%s\033[0m\n",
-                    g_Dbs.db[i].superTbls[j].stbName);
-
-            if (PRE_CREATE_SUBTBL == g_Dbs.db[i].superTbls[j].autoCreateTable) {
-                printf("      autoCreateTable:   \033[33m%s\033[0m\n",  "no");
-            } else if (AUTO_CREATE_SUBTBL ==
-                    g_Dbs.db[i].superTbls[j].autoCreateTable) {
-                printf("      autoCreateTable:   \033[33m%s\033[0m\n",  "yes");
-            } else {
-                printf("      autoCreateTable:   \033[33m%s\033[0m\n",  "error");
-            }
-
-            if (TBL_NO_EXISTS == g_Dbs.db[i].superTbls[j].childTblExists) {
-                printf("      childTblExists:    \033[33m%s\033[0m\n",  "no");
-            } else if (TBL_ALREADY_EXISTS == g_Dbs.db[i].superTbls[j].childTblExists) {
-                printf("      childTblExists:    \033[33m%s\033[0m\n",  "yes");
-            } else {
-                printf("      childTblExists:    \033[33m%s\033[0m\n",  "error");
-            }
-
-            printf("      childTblCount:     \033[33m%"PRId64"\033[0m\n",
-                    g_Dbs.db[i].superTbls[j].childTblCount);
-            printf("      childTblPrefix:    \033[33m%s\033[0m\n",
-                    g_Dbs.db[i].superTbls[j].childTblPrefix);
-            printf("      dataSource:        \033[33m%s\033[0m\n",
-                    g_Dbs.db[i].superTbls[j].dataSource);
-            printf("      iface:             \033[33m%s\033[0m\n",
-                    (g_Dbs.db[i].superTbls[j].iface==TAOSC_IFACE)?"taosc":
-                    (g_Dbs.db[i].superTbls[j].iface==REST_IFACE)?"rest":"stmt");
-            if (g_Dbs.db[i].superTbls[j].childTblLimit > 0) {
-                printf("      childTblLimit:     \033[33m%"PRId64"\033[0m\n",
-                        g_Dbs.db[i].superTbls[j].childTblLimit);
-            }
-            if (g_Dbs.db[i].superTbls[j].childTblOffset > 0) {
-                printf("      childTblOffset:    \033[33m%"PRIu64"\033[0m\n",
-                        g_Dbs.db[i].superTbls[j].childTblOffset);
-            }
-            printf("      insertRows:        \033[33m%"PRId64"\033[0m\n",
-                    g_Dbs.db[i].superTbls[j].insertRows);
-            /*
-               if (0 == g_Dbs.db[i].superTbls[j].multiThreadWriteOneTbl) {
-               printf("      multiThreadWriteOneTbl:  \033[33m no\033[0m\n");
-               }else {
-               printf("      multiThreadWriteOneTbl:  \033[33m yes\033[0m\n");
-               }
-               */
-            printf("      interlaceRows:     \033[33m%u\033[0m\n",
-                    g_Dbs.db[i].superTbls[j].interlaceRows);
+        if (g_args.use_metric)
```

Comment:
please keep the same coding style as following:

if (cond) {
do something
} else {
do something else
}